src/test/java/com/epam/tc/hw3/testcases/Exercise2.java

```diff
@@ -0,0 +1,107 @@
+package com.epam.tc.hw3.testcases;
+
+import com.epam.tc.hw3.library.pages.DifElementsPage;
+import com.epam.tc.hw3.library.pages.Header;
+import com.epam.tc.hw3.library.utils.GetProperties;
+import java.time.Duration;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class Exercise2 extends BaseTest {
+
+    //1. Open test site by URL - in parent class
+    @Test
+    public void exercise2() {
+        //2. Assert Browser title
+        SoftAssertions softly = new SoftAssertions();
+        String browserTitle = testDriver.getTitle();
+        softly.assertThat(browserTitle).as("Incorrect browser title").isEqualTo("Home Page");
+
+        //3. Perform login
+        Header header = PageFactory.initElements(testDriver, Header.class);
```

Comment:
Why are you using `PageFactory.initElements` instead of constructor?

Reply:
Fixed